doc/source/integration.rst

```diff
@@ -100,18 +100,19 @@ Typically, the resulting code looks like this:
 
 .. code:: python
 
-   import ansys.platform.instancemanagement as pypim
+    import ansys.platform.instancemanagement as pypim
+
 
-   def launch_my_product(self, ...):
-       if pypim.is_configured() and not user_has_specified_how_to_launch_the_product:
-           pim = pypim.connect()
-           self.instance = pim.create_instance("my_product_name")
-           self.instance.wait_for_ready()
-           channel = self.instance.build_grpc_channel()
-       else:
-           # usual start-up
-           self.process = subprocess.run(...)
-           channel = grpc.insecure_chanel(...)
+    def launch_my_product():
+        if pypim.is_configured():
+            pim = pypim.connect()
+            self.instance = pim.create_instance("my_product_name")
+            self.instance.wait_for_ready()
+            channel = self.instance.build_grpc_channel()
+        else:
+            # usual start-up
+            self.process = subprocess.run(...)
+            channel = grpc.insecure_chanel(...)
```

Comment:
@plule-ansys Could you please try to fix this code block? Blacken-docs, a linter declared within pre-commit complains about it. I bet the reason is that this is supposed to be a class method but it is not declared inside a class.

Reply:
Fixed! (it was a missing newline between code-block and the code)